model/task/task_test.go

```diff
@@ -1202,3 +1202,16 @@ func TestGetTimeSpent(t *testing.T) {
 	assert.EqualValues(0, timeTaken)
 	assert.EqualValues(0, makespan)
 }
+
+func TestSetAborted(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
I think in general we should *not* do this, and instead just pass t to every assert. 

also the assert on 1216 will panic if 1215 fails, which means we should probably require there.

Reply:
done.